roles/pulp_redis/handlers/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: restart redis
   systemd:
-    name: "{{ pulp_redis_server_name }}"
+    name: "{{ pulp_redis_server_name | default(_pulp_redis_server_name) }}"
```

Comment:
Why the default here? the value is already in `defaults/main.yaml`

Reply:
@mdellweg The server name is not part of `defaults/main.yml` (Because it is OS specific), only `pulp_redis_package_name` is hence why the default pattern is being used here.